spec/atalogics_api/v2/client_spec.rb

```diff
@@ -200,10 +200,10 @@
     let(:body) {{
       addresses: [
         {lat: 47.8065258, lng: 13.0474424, city_key: "SALZBURG"},
-        {street: "Radetzkystrasse", number: "7", postal_code: 5020, city: "Salzburg", city_key: "SALZBURG"}
+        {street: "Radetzkystrasse", number: "7", postal_code: 5020, city: "Salzburg", city_key: "SALZBURG", country: "AT"}
       ]
     }}
-    let(:cache_key) { "V2___/addresses/multi/check_SALZBURG_____47.8065258_13.0474424_SALZBURG_Radetzkystrasse_7_5020_Salzburg__" }
+    let(:cache_key) { "V2___/addresses/multi/check_SALZBURG______47.8065258_13.0474424_SALZBURG_Radetzkystrasse_7_5020_Salzburg_AT__" }
```

Comment:
Wieso kommt hier nach SALZBURG ein Underscore dazu?

Reply:
wegen der Änderung in der v2/client.rb, es kommt ja bei jedem Adress Check eine Component dazu (das Country), also gibts auch einen _ mehr wenn ich nach Lat/Lng suche

Reply:
Ah, etwas verwirrend, weil im `key` nochmal Lat und Lng dabei sind und im `cache_key` dann auch nochmal